common/dpsl.ts

```diff
@@ -77,10 +77,10 @@ export interface CpuCStateInfo {
  */
 export interface LogicalCpuInfo {
   maxClockSpeedKhz: number;
-  ScalingMaxFrequencyKhz: number;
-  ScalingCurrentFrequencyKhz: number;
-  IdleTimeMs: number;
-  AcStates: CpuCStateInfo[];
+  scalingMaxFrequencyKhz: number;
+  scalingCurrentFrequencyKhz: number;
+  idleTimeMs: number;
+  acStates: CpuCStateInfo[];
```

Comment:
Those fields were refactored to make the GSOC demo work. I can move them to another PR, but I prefer to keep it here. WDYT?

Reply:
I am OK to make such changes in this CL.